src/DataStructures/Python/Matrix.hpp

```diff
@@ -0,0 +1,10 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#pragma once
+
+#include "DataStructures/Matrix.hpp"
```

Comment:
You don't need either of these `include` statements because in the header you are not using either class